src/motion/index.tsx

```diff
@@ -12,7 +12,6 @@ import { useMotionRef } from "./utils/use-motion-ref"
 import { useCreateMotionContext } from "../context/MotionContext/create"
 import { loadFeatures } from "./features/definitions"
 import { isBrowser } from "../utils/is-browser"
-export { MotionProps }
```

Comment:
Would this change also influence imports like this?

```typescript
import { MotionProps } from 'framer-motion'
```

If so, it could be good to add a note about this to the changelog as well.

Reply:
It shouldn’t, are you seeing any?